rubocop-sorbet.gemspec

```diff
@@ -1,31 +1,31 @@
 # frozen_string_literal: true
-
-lib = File.expand_path('../lib', __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require_relative "lib/rubocop/sorbet/version"
```

Comment:
Minor nit: I think we should consistently use either single quotes or double quotes. IIRC the `rubocop-shopify`  gem does not have a preference.

Reply:
Indeed and that's why I'd changed all usages in `.gemspec` to use double-quotes. It should be consistent. Am I missing something?

Reply:
I think there's a mix of the two in places like [`here`](https://github.com/Shopify/rubocop-sorbet/blob/5c717f454f867dfa06b7ea9b4518169ddbb0f2ea/Gemfile#L10), [`here`](https://github.com/Shopify/rubocop-sorbet/blob/5c717f454f867dfa06b7ea9b4518169ddbb0f2ea/spec/spec_helper.rb#L3), or [`here`](https://github.com/Shopify/rubocop-sorbet/blob/5c717f454f867dfa06b7ea9b4518169ddbb0f2ea/lib/rubocop/cop/sorbet_cops.rb#L2).

Reply:
Ok, resolved some of them, but I am not going to change it all across the codebase if usages within a single file are consistent. For example, your last example.

Reply:
:+1: Thanks!